flottform/forms/src/internal.ts

```diff
@@ -16,6 +16,20 @@ type EndpointInfo = {
 		iceCandidates: RTCIceCandidateInit[];
 	};
 };
+
+export type BaseListeners = {
+	new: [];
+	disconnected: [];
+	error: [error: any];
+	connected: [];
+	'endpoint-created': [{ link: string; qrCode: string }];
+	'webrtc:waiting-for-client': [
+		event: { link: string; qrCode: string; channel: FlottformChannelHost }
+	];
+	'webrtc:waiting-for-ice': [];
+	'webrtc:waiting-for-file': [];
```

Comment:
I think we need to remove it from here and then add the event that best fits the class.
For example inside `FlottformTextInputHost` we can add `webrtc:waiting-for-text` and for `FlottformFileInputHost` we can add `webrtc:waiting-for-file`
```suggestion
```